src/main/java/frc/robot/subsystems/drivetrain/commands/DriveAndAdjustWithVisionExperimental.java

```diff
@@ -0,0 +1,156 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.Constants;
+import frc.robot.Robot;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+import static frc.robot.Constants.SwerveDrive.VELOCITY_MULTIPLIER;
+
+public class DriveAndAdjustWithVisionExperimental extends HolonomicDrive {
```

Comment:
Why not just extend from OverpoweredDrive and have most of it implemented?

Reply:
I don't see how I would go about on doing that, and I believe it would only complicate things.